example-ladle/lostpixel.config.ts

```diff
@@ -2,7 +2,7 @@ import { CustomProjectConfig } from "lost-pixel";
 
 export const config: CustomProjectConfig = {
   ladleShots: {
-    ladleUrl: "http://172.17.0.1:61000",
+    ladleUrl: "http://127.0.0.1:61000",
```

Comment:
this will be true for local run, but on github actions it will fail as there they have a specific IP

Reply:
Ok, but then it won't work locally.

```suggestion
    ladleUrl: "http://172.17.0.1:61000",
```

Reply:
yeah, that could be improved somehow. Will think about how we can educate better about it